### ISSUE TITLE
Metro PDF Merger worker failed silently

### ISSUE THREAD

**Issue by fgregg** [datamade/la-metro-councilmatic#738]
The metro pdf merger worker failed because of an out of memory condition due to transient memory pressure on the server.

Because of that failure, board packets have not been building for a few days, which was finally noticed by @shrayshray today.

Suggested mitigation:

1. Update the supervisor settings to automatically attempt to restart the worker
2. Make sure that this type of failure alerts DataMade

**Comment by fgregg** [comment]
It may also be worth considering a different data flow, wherein after a scraping job task finishes, airflow then picks up a compile_pdfs tasks that runs a new `lametro` management command that actually runs the compiling code that currently lives in `metro-pdf-merger`, and [saves the files in a S3 File store](https://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html). That management command would have to run in a docker instance that had the system dependencies.

This would remove metro-pdf-merger completely, both as the place where pdf merges happen but also as the server for serving files. 

We could do something similar for the files we are currently mirroring on pic.datamade.us, and have an airflow job and management command take reponsibility for putting those files in s3 storage.

**Comment by fgregg** [comment]
This consolidation would reduce the footprint of the application considerably, and bring all the data tasks into the airflow framework.

**Comment by fgregg** [comment]
the worker failed again. Here's the traceback:

```bash
[June 18, 2021 - 19:52:31][ERROR] tasks.py:line:120 | <class 'SystemExit'> cause
d the failure of writing regular-board-meeting-c62a8c6bcd17 as a PDF, and we cou
ld not merge this file collection: 
 ['https://metro.legistar1.com/metro/meetings/2021/2/1942_A_Board_of_Directors_-Regular_Board_Meeting_21-02-25_Agenda.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=7408&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/0f654c0c-1e65-42c4-a503-2bda2b82c566.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=6917&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/8cf422b4-e7aa-4be1-bfb4-d1d0ef4d6c54.pdf', 'http://metro.legistar1.com/metro/attachments/e95501f3-3347-4286-ba5c-eae3511a215c.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=7350&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/9c189c7e-d194-405e-94b7-05106c74adc0.pdf', 'http://metro.legistar1.com/metro/attachments/76b413c3-90df-4e06-9505-28cb936ccacc.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=7209&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/bbf88548-a342-4904-bae2-e9f17f81560f.pdf', 'http://metro.legistar1.com/metro/attachments/d94368c3-0d28-4a45-a818-20215c033594.pdf', 'http://metro.legistar1.com/metro/attachments/a9b774c8-3d9c-4ca9-aadd-696bb3dac2a9.pdf', 'http://metro.legistar1.com/metro/attachments/1814dd24-758a-4ee3-a91e-99592a3f562d.pdf', 'http://metro.legistar1.com/metro/attachments/0fd9c50d-4668-4639-8c63-3bb23d73458c.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=7276&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/818ba340-bd7d-46f1-a12e-6e2577251b43.pdf', 'http://metro.legistar1.com/metro/attachments/55f50389-2e34-457c-aa35-82585b5249d2.pdf', 'http://metro.legistar1.com/metro/attachments/9631a0ce-0e6b-44ca-866e-cb8366f2b45d.pdf', 'http://metro.legistar1.com/metro/attachments/dd83468e-a85e-4dc2-8744-f70211943dc2.pdf', 'http://metro.legistar1.com/metro/attachments/f83aee3b-cf39-4254-a5e7-48b6952ad9d7.pdf', 'http://metro.legistar1.com/metro/attachments/b858ac31-1a95-4004-ab15-4497a1bad0fa.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=7293&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/c2bfb0c3-a741-4bfa-9ca6-3d06988903e2.pdf', 'http://metro.legistar1.com/metro/attachments/8a961f6f-5bc0-4482-bbbc-13ec124bc88c.pdf', 'http://metro.legistar1.com/metro/attachments/b00c74d1-87ac-4522-99f0-f04e659b37c4.pdf', 'http://metro.legistar1.com/metro/attachments/4b1ae8d3-acc3-48b3-934c-26f9a2a16e29.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=7234&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/10309abe-a44b-4157-8498-be4a59dcaed6.pdf', 'http://metro.legistar1.com/metro/attachments/aa8d23f1-b737-4214-b07c-8ce89f95c7ca.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=7282&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/167fbddd-c73f-4d44-aef4-0a895b178a89.pdf', 'http://metro.legistar1.com/metro/attachments/2f447a58-bbae-4827-a092-c6365ba9b565.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=7285&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/6fc8b65a-bafb-4817-821b-21818fb2fdd4.pdf', 'http://metro.legistar1.com/metro/attachments/50356d58-7430-4dfc-8fb3-3a0c44cb2c8b.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=7307&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/8f06923b-6354-4f22-bffa-f2b18fa3a931.pdf', 'http://metro.legistar1.com/metro/attachments/f85c55f9-cbfc-4a5e-8f35-81f4a3f24b51.pdf', 'http://metro.legistar1.com/metro/attachments/5505332b-cbdd-4ad2-81cc-cf417b9ea106.pdf', 'http://metro.legistar1.com/metro/attachments/215aded1-3240-481e-8aed-a901ddcf5258.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=7316&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/14ee5b23-5d18-4d88-b8c0-da0950bc4d32.pdf', 'http://metro.legistar1.com/metro/attachments/cf1ca491-39f8-45a6-88de-8ddd41c8c736.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=7184&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/b7cc06f0-7f41-44c9-84ad-e5c01c0f74e3.pdf', 'http://metro.legistar1.com/metro/attachments/192506e9-3929-4621-9d19-6d33dc46bd59.pdf', 'http://metro.legistar1.com/metro/attachments/51e5e6a4-c4d1-4046-b5de-53176c84cc43.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=7335&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/c39a981a-0396-4633-8e10-2983c8016c8b.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=7327&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/24a10747-7a93-4bbd-b1d9-3b830dd59692.pdf', 'http://metro.legistar1.com/metro/attachments/7d3a7c71-ba6f-4141-bbcb-05aacd706fe9.pdf', 'http://metro.legistar1.com/metro/attachments/aae18d64-379a-41a4-8fee-cdcca8748777.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=7318&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/24b7aca4-5a6f-4f76-97d8-1e2b2077ee60.pdf', 'http://metro.legistar1.com/metro/attachments/38f63803-259c-4cef-9eb2-74e3ad38f30b.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=7422&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/2ea3a169-9918-4c6a-877f-80f87a129a20.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=7289&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/687a0540-a3af-4584-a2e2-0852b39d970d.pdf', 'http://metro.legistar1.com/metro/attachments/98c1af2a-ff09-4d37-aaf2-f13f1099b1cd.pdf', 'http://metro.legistar1.com/metro/attachments/83446543-c44c-409e-9b48-27376c138bea.pdf', 'http://metro.legistar1.com/metro/attachments/6c5e1d5a-d67c-4d33-95dd-56e664a154ad.pdf', 'http://metro.legistar1.com/metro/attachments/7e570676-3073-4900-8b10-7381dcb17109.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=7376&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/13dfd102-a85c-4f25-b88d-b954d3dd49b7.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=6496&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/0c31a57b-c5d7-48a4-a5f7-55945ba5f83e.pdf', 'http://metro.legistar1.com/metro/attachments/af20da38-ce5b-4ab3-bb72-c668704e7a40.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=7330&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/f7852e36-0a6a-41bc-9539-77f9dee951d6.pdf', 'http://metro.legistar1.com/metro/attachments/c247a5fc-6b94-461b-aefb-2310a66de375.pdf', 'http://metro.legistar1.com/metro/attachments/3a63642d-5d74-4847-843e-e6969bebac61.pdf', 'http://metro.legistar1.com/metro/attachments/1ac7472c-0cef-4ab0-9f28-ca2d53c8416b.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=7331&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/e8a96621-9dea-4896-be93-28eb18571d14.pdf', 'http://metro.legistar1.com/metro/attachments/b55487a2-a537-4dbb-93be-292463f1eb72.pdf', 'http://metro.legistar1.com/metro/attachments/cb864dfe-1982-4e9f-908e-67e6a5386056.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=7399&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/91d3eb34-3bee-4b50-8058-85ff557908f8.pdf', 'http://metro.legistar1.com/metro/attachments/81aedf63-90ba-47b4-933d-628402d05166.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=7355&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/38f00a5f-e8ea-4d63-909d-42053489b465.pdf', 'https://metro.legistar.com/ViewReport.ashx?M=R&N=TextL5&GID=557&ID=7412&GUID=LATEST&Title=Board+Report', 'http://metro.legistar1.com/metro/attachments/5a65ba19-53ed-47ac-b008-ad72521903b1.pdf']
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/threading.py", line 914, in _bootstrap_inner
    self.run()
  File "/home/datamade/metro-pdf-merger-d-L142C7MP9/tasks.py", line 145, in run
    self.doWork()
  File "/home/datamade/metro-pdf-merger-d-L142C7MP9/tasks.py", line 151, in doWork
    child.start()
  File "/usr/local/lib/python3.5/multiprocessing/process.py", line 105, in start
    self._popen = self._Popen(self)
  File "/usr/local/lib/python3.5/multiprocessing/context.py", line 212, in _Popen
    return _default_context.get_context().Process._Popen(process_obj)  File "/usr/local/lib/python3.5/multiprocessing/context.py", line 267, in _Popen
    return Popen(process_obj)
  File "/usr/local/lib/python3.5/multiprocessing/popen_fork.py", line 20, in __init__
    self._launch(process_obj)
  File "/usr/local/lib/python3.5/multiprocessing/popen_fork.py", line 67, in _launch
    self.pid = os.fork()
OSError: [Errno 12] Cannot allocate memory
``` 

it's likely that pulling together all these pdfs simply exhausted the memory on the server. we'll have to pick this up next week.

**Comment by hancush** [comment]
tl;dr - I've restarted The Merger with some improvements. We're babysitting; if it fails again, it will help to direct our next steps. Barring another failure, we should start seeing packets for recent events and board reports over the next few days.

---

I've corrected an error in logging configuration, added more granular logging, and updated the code to raise non-zero exit codes when a child process dies. We are still investigating why the parent worker process (i.e., The Merger) dies at the same time as its child (we suspect [the dreaded OOM Killer](https://www.instana.com/blog/solving-the-out-of-memory-killer-puzzle/)), but I've restarted it to kick the tires of these changes and, in the event of another death, give us recent signal to follow (as logs from June have regretfully left the server).

**Comment by hancush** [comment]
Update: I stopped the merger, as we started running up against memory constraints which caused the main Board Agendas site to run sluggishly, not good on a meeting day. 

Downloaded the logs, attached here. Seems like we couldn't allocate memory for `unoconv`: 

```
[September 16, 2021 - 16:09:46][INFO] tasks.py:line:68 | Unoconv conversion underway...
Process ChildProcessor-650:
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/multiprocessing/process.py", line 254, in _bootstrap
    self.run()
  File "/home/datamade/metro-pdf-merger-d-PU86DN17D/tasks.py", line 143, in run
    func(*args, **kwargs)
  File "/home/datamade/metro-pdf-merger-d-PU86DN17D/tasks.py", line 69, in makePacket
    check_output(['unoconv', '-f', 'pdf', filename])
  File "/usr/local/lib/python3.5/subprocess.py", line 629, in check_output
    **kwargs).stdout
  File "/usr/local/lib/python3.5/subprocess.py", line 696, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/usr/local/lib/python3.5/subprocess.py", line 950, in __init__
    restore_signals, start_new_session)
  File "/usr/local/lib/python3.5/subprocess.py", line 1483, in _execute_child
    restore_signals, start_new_session, preexec_fn)
OSError: [Errno 12] Cannot allocate memory
```

– and later, like we couldn't start a new thread –

```
Process ChildProcessor-653:
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/multiprocessing/process.py", line 254, in _bootstrap
    self.run()
  File "/home/datamade/metro-pdf-merger-d-PU86DN17D/tasks.py", line 143, in run
    func(*args, **kwargs)
  File "/home/datamade/metro-pdf-merger-d-PU86DN17D/tasks.py", line 128, in makePacket
    s3_key.upload_fileobj(merged)
  File "/home/datamade/.virtualenvs/metro-pdf-merger-d-PU86DN17D/lib/python3.5/site-packages/boto3/s3/inject.py", line 513, in object_upload_fileobj
    ExtraArgs=ExtraArgs, Callback=Callback, Config=Config)
  File "/home/datamade/.virtualenvs/metro-pdf-merger-d-PU86DN17D/lib/python3.5/site-packages/boto3/s3/inject.py", line 430, in upload_fileobj
    extra_args=ExtraArgs, subscribers=subscribers)
  File "/home/datamade/.virtualenvs/metro-pdf-merger-d-PU86DN17D/lib/python3.5/site-packages/s3transfer/manager.py", line 308, in upload
    call_args, UploadSubmissionTask, extra_main_kwargs)
  File "/home/datamade/.virtualenvs/metro-pdf-merger-d-PU86DN17D/lib/python3.5/site-packages/s3transfer/manager.py", line 464, in _submit_transfer
    main_kwargs=main_kwargs
  File "/home/datamade/.virtualenvs/metro-pdf-merger-d-PU86DN17D/lib/python3.5/site-packages/s3transfer/futures.py", line 435, in submit
    future = ExecutorFuture(self._executor.submit(task))
  File "/usr/local/lib/python3.5/concurrent/futures/thread.py", line 113, in submit
    self._adjust_thread_count()
  File "/usr/local/lib/python3.5/concurrent/futures/thread.py", line 129, in _adjust_thread_count
    t.start()
  File "/usr/local/lib/python3.5/threading.py", line 844, in start
    _start_new_thread(self._bootstrap, ())
RuntimeError: can't start new thread
```

We didn't get [any errors](https://sentry.io/organizations/datamade/issues/?project=155211), which sucks, and interestingly, the parent process did not die – I stopped it with Supervisor.

📄 [merger.log](https://github.com/datamade/la-metro-councilmatic/files/7179488/merger.log)

**Comment by fgregg** [comment]
hmm.. do we need to anything to initialize sentry in the children?

**Comment by hancush** [comment]
@fgregg Likely.

Some evidence of the OOM Killer: 

```
ubuntu@ip-10-0-0-208:~$ grep kill /var/log/syslog
Sep 16 16:12:47 ip-10-0-0-208 kernel: [7513660.424375] supervisord invoked oom-killer: gfp_mask=0x201da, order=0, oom_score_adj=0
Sep 16 16:12:47 ip-10-0-0-208 kernel: [7513660.424416]  [<ffffffff8115b861>] oom_kill_process+0x201/0x360
Sep 16 16:22:55 ip-10-0-0-208 kernel: [7514268.224306] python invoked oom-killer: gfp_mask=0x280da, order=0, oom_score_adj=0
Sep 16 16:22:55 ip-10-0-0-208 kernel: [7514268.224350]  [<ffffffff8115b861>] oom_kill_process+0x201/0x360
```

It seems like the worker restarted!!!!

```
call-me-hank:merge-memory-test hannah$ ssh ubuntu@metro-pdf-merger.datamade.us
Welcome to Ubuntu 14.04.5 LTS (GNU/Linux 3.13.0-170-generic x86_64)

 * Documentation:  https://help.ubuntu.com/

 System information disabled due to load higher than 2.0

  Get cloud support with Ubuntu Advantage Cloud Guest:
    http://www.ubuntu.com/business/services/cloud

63 packages can be updated.
0 updates are security updates.

New release '16.04.7 LTS' available.
Run 'do-release-upgrade' to upgrade to it.


Last login: Thu Sep 16 16:15:12 2021 from 174-086-246-086.res.spectrum.com
ubuntu@ip-10-0-0-208:~$ sudo supervisorctl status
-bash: fork: Cannot allocate memory
ubuntu@ip-10-0-0-208:~$ sudo supervisorctl status
...
pdf-merger-d-PU86DN17D:metro-pdf-merger               RUNNING   pid 9903, uptime 2:09:10
metro-pdf-worker-d-PU86DN17D:metro-pdf-worker               RUNNING   pid 6213, uptime 0:02:28
...
```

Note the difference in app/worker uptime; and the timestamp on my server login – the uptime would fit with the narrative that the OOM Killer took out the supervisor process @ 16:12.

But I'm starting to feel like restarting the worker w/o resolving the memory issue is probably not a feasible path forward, as it just picks up gobbling memory where it leaves off...

**Comment by hancush** [comment]
Confirmation of worker restart:

```
ubuntu@ip-10-0-0-208:~$ tail -n 50 /var/log/supervisor/supervisord.log
...
# Logs from creation of processes via deployment
2021-09-16 14:13:40,978 INFO spawned: 'metro-pdf-merger' with pid 9903
2021-09-16 14:13:41,172 INFO spawned: 'metro-pdf-worker' with pid 9905
2021-09-16 14:13:42,485 INFO success: metro-pdf-worker entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2021-09-16 14:13:42,485 INFO success: metro-pdf-merger entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)

# This timestamp lines up with OOM Killer, interesting we don't see a SIGABRT until 7 minutes later......
2021-09-16 14:13:44,004 INFO exited: metro-pdf-merger (exit status 0; expected)
2021-09-16 16:20:22,343 INFO exited: metro-pdf-worker (terminated by SIGABRT (core dumped); not expected)
2021-09-16 16:20:22,377 INFO spawned: 'metro-pdf-worker' with pid 6213
2021-09-16 16:20:23,471 INFO success: metro-pdf-worker entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)

# Logs corresponding to manual shutdown via supervisorctl stop
2021-09-16 16:23:19,066 INFO waiting for metro-pdf-worker to stop
2021-09-16 16:23:21,066 INFO waiting for metro-pdf-worker to stop
2021-09-16 16:23:23,068 INFO waiting for metro-pdf-worker to stop
2021-09-16 16:23:25,068 INFO waiting for metro-pdf-worker to stop
2021-09-16 16:23:27,068 INFO waiting for metro-pdf-worker to stop
2021-09-16 16:23:29,066 WARN killing 'metro-pdf-worker' (6213) with SIGKILL
2021-09-16 16:23:29,068 INFO waiting for metro-pdf-worker to stop
2021-09-16 16:23:29,068 INFO stopped: metro-pdf-worker (terminated by SIGKILL)
```

**Comment by fgregg** [comment]
so, lots of different options here. i suppose the next question is whether it is acceptable for some jobs to not complete because they are going to take too much memory.

if the answer is no, we need to move to another architecture. if we the answer is yes, or at least, it's preferable than *no* jobs being completed, we could modify our queuing set up so that we will retry a job a few times, and if it doesn't succeed then we'll skip it.

**Comment by hancush** [comment]
More OOM Killer deetz:

```
ubuntu@ip-10-0-0-208:~$ grep -i "out of memory" /var/log/kern.log
Sep 16 16:12:47 ip-10-0-0-208 kernel: [7513660.424836] Out of memory: Kill process 18163 (soffice.bin) score 530 or sacrifice child
Sep 16 16:22:55 ip-10-0-0-208 kernel: [7514268.224778] Out of memory: Kill process 18163 (soffice.bin) score 530 or sacrifice child
```

**Comment by fgregg** [comment]
TIL about SIGABRT

**Comment by hancush** [comment]
Last week, I moved the merger to an existing server that's twice as powerful as its previous home, then manually triggered packet creation for all events. It handled that load without any issues, so I've updated the site and dashboard to interact with the beefier merger instance. I also improved our logging implementation, so we should receive events from failures.

**Comment by hancush** [comment]
Update: The beefier merger chewed through all available memory in under an hour. Starting to feel like it might be a better investment of time to consider a revised architecture, than to continue trying to wrangle the inefficiencies of the current process.

**Comment by hancush** [comment]
I'll update this issue with potential next steps / estimates by EOW.

**Comment by hancush** [comment]
Want to flag that we have 120+ successful merges this past week, covering the latest round of meetings and related board reports. (I checked Friday night when agendas posted and noticed that the event packets had compiled on the staging site!) 

I am feeling quite good about it. @shrayshray, can you and @camachoo review a sample of packets from this week's meetings and their agenda items and confirm everything looks above board?

**Comment by shrayshray** [comment]
@hancush I've looked through a handful and they all looked good to me!

**Comment by hancush** [comment]
The new merger has been deployed! I've spun off two housekeeping issues, but packets should start coming in as board reports and events are updated.
